include/os/linux/zfs/sys/trace_acl.h

```diff
@@ -119,7 +117,7 @@ DECLARE_EVENT_CLASS(zfs_ace_class,
 	    "zn_prefetch %u blksz %u seq %u "
 	    "mapcnt %llu size %llu pflags %llu "
 	    "sync_cnt %u sync_writes_cnt %u async_writes_cnt %u "
-	    "mode 0x%x is_sa %d is_mapped %d is_ctldir %d "
+	    "mode 0x%x is_sa %d is_ctldir %d "
```

Comment:
Do we just want to remove `is_mapped` here? If we wanted to we could do a `#if !defined(HAVE_FILEMAP_RANE_HAS_PAGE)` guards to keep this here in the event we still are using `z_is_mapped`.

Reply:
I thought about that be in the end decided against it to keep the trace point clean since for any semi-modern kernel `HAVE_FILEMAP_RANE_HAS_PAGE` will be defined.  It seems unlikely we'll ever be debugging a very old kernel and want to see that state.